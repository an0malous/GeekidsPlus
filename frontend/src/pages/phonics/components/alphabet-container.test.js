import React from 'react';
import { shallow } from 'enzyme';
import AlphabetContainer from './alphabet-container.component';
import configureStore from 'redux-mock-store';
import { Provider } from 'react-redux';
import renderer from 'react-test-renderer';

const mockStore = configureStore([]);

describe('My Connected React-Redux Component', () => {
	let store;
	let component;

	beforeEach(() => {
		store = mockStore({
			phonicsGameReducer: {
            currentWords: [ {name: "push", type: "blends", audio: null, img: null}],
				currentDeckIndex: 0
			},
		});

		component = renderer(
			<Provider store={store}>
				<AlphabetContainer />
			</Provider>
		);
	});

	it('should render with given state from Redux store', () => {
		expect(component).toMatchSnapShot();
   });

})
