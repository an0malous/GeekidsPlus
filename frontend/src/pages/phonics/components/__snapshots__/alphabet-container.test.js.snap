// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Connected React-Redux Component should render with given state from Redux store 1`] = `
<div
  className="ui grid"
>
  <div
    className="centered row"
  >
    Loading Alphabet Cards...
  </div>
</div>
`;
